src/main/java/com/karankumar/bookproject/backend/entity/BaseEntity.java

```diff
@@ -30,14 +30,8 @@
     @Id
     @GeneratedValue(strategy = GenerationType.SEQUENCE)
     @Getter
-    private Long id;
+    private Long id = null;
```

Comment:
Do we need explicit null assignment?

Reply:
we can remove it

Reply:
@NyorJa, I noticed you marked this conversation as resolved. However, I think it would be better for a reviewer (Amit or myself) to do so when we can confirm that this has been changed (as per our contributing guidelines).